packages/nx-plugin/src/generators/app/generator.ts

```diff
@@ -51,24 +25,25 @@ export interface NormalizedOptions
   projectDirectory: string;
   parsedTags: string[];
   offsetFromRoot: string;
+  appsDir: string;
+  nxPackageNamespace: string;
 }
 
 function normalizeOptions(
   tree: Tree,
-  options: AnalogNxApplicationGeneratorOptions
+  options: AnalogNxApplicationGeneratorOptions,
+  nxVersion: string
 ): NormalizedOptions {
+  const appsDir = getWorkspaceLayout(tree).appsDir;
   const allNames = names(options.name);
-  const name = allNames.fileName;
-  const projectDirectory = options.directory
-    ? `${names(options.directory).fileName}/${name}`
-    : name;
+  const projectDirectory = allNames.fileName;
   const projectName = projectDirectory.replace(new RegExp('/', 'g'), '-');
-  const projectRoot = `${getWorkspaceLayout(tree).appsDir}/${projectDirectory}`;
+  const projectRoot = `${appsDir}/${projectDirectory}`;
   const parsedTags = options.tags
     ? options.tags.split(',').map((s) => s.trim())
     : [];
   const offsetFromRoot = determineOffsetFromRoot(projectRoot);
-
+  const nxPackageNamespace = major(nxVersion) >= 16 ? '@nx' : '@nrwl';
```

Comment:
Let's just use `@nx` as we'll pin everything to Nx 16.1.0 to support Angular >= v15

Reply:
The one thing I want to mention is that this is version comes from the user's Nx workspace. If I understand this correctly removing this logic would mean that we do not support workspaces that have an Nx version of 15

Reply:
Good point. Let's keep them separate

Reply:
I tested with a v15.9.2 workspace and the current  approach & versions for V15 worked 👍